lesson_04/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" type="text/css" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+</head>
+<body>
+  <div>
+    <ul id="js-target"></ul>
```

Comment:
Q.
Why did you name this id "target"?
If you have a reason, I would like to know it.
I thought it would be better to have a name that you could associate with being a list.

Reply:
Sorry. I don't mean anything in particular, I'll be careful how I put my ID.